casanovo/casanovo.py

```
@@ -633,9 +619,7 @@ def _setup_output(
         for output files.
     """
     if output_root is None:
-        output_root = (
-            f"casanovo_{datetime.datetime.now().strftime('%Y%m%d%H%M%S')}"
-        )
+        output_root = f"casanovo_{datetime.datetime.now().strftime('%Y%m%d%H%M%S')}"
```

Ruff (0.15.7)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)